spiffworkflow-backend/tests/spiffworkflow_backend/unit/test_background_processing_service.py

```
@@ -35,6 +38,15 @@ def run(
     ) -> dict[str, Any]: ...
 
 
+class SupportsCeleryStartFromModelTaskRun(Protocol):
+    def run(
```

Flake8 (7.3.0)

[error] multiple statements on one line (def)

(E704)